apps/sms/views/conversation/js/conversation.js

```diff
@@ -2676,7 +2675,7 @@ var ConversationView = {
       return;
     }
 
-    if (!contacts || !contacts.length) {
```

Comment:
I changed the return value to empty array for ConversationView.searchContact
conversation_test.js  & conversation_integration_test.js give no errors after that.
Do i need to check for somethings else?